product_supplierinfo_for_customer_sale_quick/views/product_view.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="product_tree_view" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="sale_quick.product_tree_view4sale" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree//field[@name='display_name']" position="after">
+                <field name="product_customer_code" optional="show" />
+            </xpath>
+
+            <xpath expr="//tree//field[@name='display_name']" position="after">
+                <field name="product_customer_name" optional="show" />
+            </xpath>
+
+        </field>
+    </record>
+
+    <record id="product_search_inherit" model="ir.ui.view">
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_search_form_view" />
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="product_customer_name" />
+            </field>
+            <field name="name" position="after">
+                <field name="product_customer_code" />
+            </field>
```

Comment:
**chore**: syntax can be simplified as above. See comment in model for other issues.